ml/models/logo_detection/main.py

```diff
@@ -0,0 +1,12 @@
+from process import process
```

Comment:
Same as import above
```suggestion
from .process import process
```